src/__snapshots__/compileToIntermediateTree.test.js.snap

```diff
@@ -0,0 +1,750 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should allow injecting context to elements 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "heading",
+      Object {
+        "level": 1,
+        "text": "foo",
+      },
+      null,
+    ],
+  ],
+}
+`;
+
+exports[`should allow overriding block code element 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "code",
+      Object {
+        "code": "code",
+        "language": "js",
+      },
+      null,
+    ],
+  ],
+}
+`;
+
+exports[`should be able to combine in compilation 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "heading",
+      Object {
+        "level": 1,
+        "text": "hey",
+      },
+      null,
+    ],
+    Array [
+      "ul",
+      null,
+      Array [
+        null,
+        Array [
+          "li",
+          null,
+          Array [
+            "foo",
+          ],
+        ],
+        null,
+      ],
+    ],
+    Array [
+      "p",
+      null,
+      Array [
+        "-bar",
+      ],
+    ],
+  ],
+}
+`;
+
+exports[`should be able to compile components using H and marksy language 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "code",
+      Object {
+        "code": "h(Test)",
+        "language": "marksy",
+      },
+      null,
+    ],
+  ],
+}
+`;
+
+exports[`should be able to compile headers 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "heading",
+      Object {
+        "level": 1,
+        "text": "header1",
+      },
+      null,
+    ],
+    Array [
+      "heading",
+      Object {
+        "level": 2,
+        "text": "header2",
+      },
+      null,
+    ],
+    Array [
+      "heading",
+      Object {
+        "level": 3,
+        "text": "header3",
+      },
+      null,
+    ],
+    Array [
+      "heading",
+      Object {
+        "level": 4,
+        "text": "header4",
+      },
+      null,
+    ],
+  ],
+}
+`;
+
+exports[`should be able to compile html 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "div",
+      Object {
+        "dangerouslySetInnerHTML": Object {
+          "__html": "<div>hello</div>",
+        },
+        "key": "md_0",
+      },
+      null,
+    ],
+  ],
+}
+`;
+
+exports[`should be able to compile image 1`] = `
+Object {
+  "tree": Array [
+    Array [
+      "p",
```

Comment:
Why there's a `p` tag wrapper for image?

Reply:
wrapper